webapp/services.py

```diff
@@ -136,13 +150,24 @@ def create_asset(
 
         # Save the file in Swift
         file_manager.create(file_content, url_path)
+        tags = self.create_tags_if_not_exist(tags)
+        products = self.create_products_if_not_exists(products)
+        author = self.create_author_if_not_exist(author)
 
         # Save file info in Postgres
         asset = Asset(
-            file_path=url_path, data=data, tags=[], created=datetime.utcnow()
+            file_path=url_path,
+            data=data,
+            tags=tags,
+            created=datetime.utcnow(),
```

Comment:
I get this warning for this code

![image](https://github.com/user-attachments/assets/ae179648-bfba-4d58-b51d-04bf4c0dbe13)